avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,26 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, options=None):
+        """Check packet loss that occurs during ping.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        You must have sudo permissions to run this method
```

Comment:
Only if the options being passed, requires, right? Why limiting this to sudo users?

Reply:
Modified the description as per the code modification, as described in above comments